services/algolia-sync/src/components/aws-sqs/sqs.js

```diff
@@ -0,0 +1,31 @@
+require('aws-sdk/clients/sqs');
+const AWS = require('aws-sdk/global');
+const algolia = require('../algolia/message-sync');
+const cfg = require('./config');
+require('dotenv').config();
+
+AWS.config.update({ region: 'us-east-1' });
```

Comment:
This needs to be configurable -- either with the [default](https://docs.aws.amazon.com/sdk-for-javascript/v2/developer-guide/setting-region.html) `AWS_REGION` or via a new `ALGOLA_SQS_REGION` (or similar) variable. You can define a default value of `us-east-1`.

Reply:
This does not need a region passed because the URL has the region in it. 

https://sqs.us-east-1.amazonaws.com/{accountnumber}/your sqs name